src/components/FlatmapVuer.vue

```diff
@@ -317,31 +319,42 @@
                   key="layersSelection"
                 />
               -->
+              <selections-group
+                v-if="pathways && pathways.length > 0"
+                title="Pathways"
+                labelKey="label"
```

Comment:
![image](https://github.com/ABI-Software/flatmapvuer/assets/37255664/f23210f2-7829-4393-8abc-10d26faf5159)
Do we want Alert visibility be first thing in the visibility list?

Reply:
I think the fCCB decided that it is important to make sure people are aware of the alert.